assets/js/googlesitekit/notifications/register-defaults.js

```diff
@@ -230,6 +231,25 @@ export const DEFAULT_NOTIFICATIONS = {
 		},
 		isDismissible: false,
 	},
+	'auth-error': {
+		Component: AuthError,
+		priority: 130,
+		areaSlug: NOTIFICATION_AREAS.ERRORS,
+		viewContexts: [
+			VIEW_CONTEXT_MAIN_DASHBOARD,
+			VIEW_CONTEXT_MAIN_DASHBOARD_VIEW_ONLY,
+			VIEW_CONTEXT_ENTITY_DASHBOARD,
+			VIEW_CONTEXT_ENTITY_DASHBOARD_VIEW_ONLY,
+			VIEW_CONTEXT_SETTINGS,
+		],
+		checkRequirements: async ( { select, resolveSelect } ) => {
+			await resolveSelect( CORE_USER ).getAuthentication();
+			const error = select( CORE_USER ).getAuthError();
```

Comment:
`getAuthError` doesn't depend on data from `getAuthentication`, so the `resolveSelect` isn't needed/relevant here.

See https://github.com/google/site-kit-wp/blob/226b8dd09322679d12ca0d5902a800d40fbc34ef/assets/js/googlesitekit/datastore/user/authentication.js#L283-L286

All selectors that depend on data from the server-side authentication state should use https://github.com/google/site-kit-wp/blob/226b8dd09322679d12ca0d5902a800d40fbc34ef/assets/js/googlesitekit/datastore/user/authentication.js#L31-L36

A quick search through the code shows that `authError` only exists on the client side which is only called here https://github.com/google/site-kit-wp/blob/226b8dd09322679d12ca0d5902a800d40fbc34ef/assets/js/googlesitekit/api/index.js#L95-L108

Reply:
Thanks @aaemnnosttv , I've since removed the call to the resolver, there are now none 👍